src/client.js

```diff
@@ -153,11 +153,29 @@ export default class Client {
    * @param {string}    [options.appVersion]                               App Version
    * @return {Promise}                                                     A promise.
    */
-  static initialize(options) {
+  static initialize() {
```

Comment:
It does go back to `init` from `initialize`. I disagree that this could cause a significant breaking change. It is much harder to go from synchronous to asynchronous. This update will be provided without introducing a breaking change.